### DISCUSSION TITLE
How to actually use this package?

### DISCUSSION THREAD

**Discussion by colivinggit** [halilcosdu/laravel-slower#25] [Q&A]
Hi,
I love the idea of this package and have been looking for a long time for this.
How do I use this?
Will it log anything automatically without installing anything in AppServiceProvider?
Where can I see logs?
I cannot see anything in route:list

**Comment by halilcosdu** [comment]
Just install as usual, execute php artisan migrate, and the records will be in the DB table called slow_logs. @colivinggit 

There is no built-in UI; I provided an example with FilamentPHP, but you should create your own visual interface.
